src/main/webapp/startup.js

```diff
@@ -26,21 +26,32 @@ const events = [
     }
 ]
 
+// TODO (remusn@) Link Google Calendar with the function
+
 /**
- * Empties the event list
- * Add the events from local storage
+ * Imports events from Google Calendar
  */
-function startup() {
+function importEvents() {
     const eventManager = new EventManager();
     for (let event of events) {
         eventManager.addEvent(event)
     }
-    
+
     let container = document.getElementsByClassName('list')[0];
     const eventRenderer = new Renderer(container);
     eventRenderer.render(eventManager.getEvents());
 }
 
+window.import = importEvents;
```

Comment:
Please make this name more detailed, you are importing it into global namespace. Also I believe you can use inner object to group all your functions.

e.g. window.itnerary.importEvents()

Reply:
It seems that I can't. I will just add "importEvents"

Reply:
try this:
window.itnerary = {} // should be done only once if it is null
window.itnerary.importEvents = importEvents;

Reply:
Thank you very much! I made the changes